README.md

```diff
@@ -154,6 +154,30 @@ kubectl delete namespace kogito-workflows
 make undeploy
 ```
 
+## Use Workflow Development Mode
+In the devmode a user can edit and reload the files using the quarkus devmode
+To enable the dev mode add the annotation
+```
+sw.kogito.kie.org/profile: dev
+```
+in the kogitoserverlessworkflow CR
+
+The external files that can be edited during the dev mode
+must be saved in a configmap per type, currently CamelRoute, OpenAPI and Async API are supported,
+to reference the configmaps in the labels section must be added using the following keys and your preferred name as values,
+the same label must be added on the configmap.
+Currently the SWF app handle Camel, OpenAPI and AsyncAPI, to handle other type is up to the user add the dependencies to handle other types
+
+```
+labels:
+.. 
+  sw.kogito.kie.org/resource-camel: mycamel-configmap
+  sw.kogito.kie.org/resource-openapi: myopenapi-configmap
+  sw.kogito.kie.org/resource-asyncapi: myasyncapi-configmap
+  sw.kogito.kie.org/resource-generic: mygeneric-configmap
+```
+Every entry in these configmaps will be turned into a file inside /src/main/resources/<type specific folder> and using the key as file name
+
```

Comment:
```suggestion
## Use Workflow Development Mode
In the development mode, a user can edit and reload the files using the quarkus dev mode.

To enable the dev mode add this annotation in the KogitoServerlessWorkflow CR:
```
  sw.kogito.kie.org/profile: dev
```

In development mode, different external files can be edited. Each file type will be saved into a different configmap.  Currently we support those types:
- CamelRoute
- OpenAPI 
- Async API
- Generic: any other type

To be able to edit a file via a configmap, you must setup a label which will reference the specific configmap:

```
  sw.kogito.kie.org/resource-camel: mycamel-configmap
  sw.kogito.kie.org/resource-openapi: myopenapi-configmap
  sw.kogito.kie.org/resource-asyncapi: myasyncapi-configmap
  sw.kogito.kie.org/resource-generic: mygeneric-configmap
```

Each label entry will be turned into a file inside the path `src/main/resources/<type specific folder>` and using the key as the filename.
```

Reply:
and GH is failing with the review comment ... Please make sure you take the overall ...